irc-against-humanity.json

```diff
@@ -19,44 +19,45 @@
 	{"text": "You can never have too much %s."},
 	{"text": "I didn't know about %s until %s introduced me to it.", "pick": 2},
 	{"text": "%s and %s. I ship it.", "pick": 2},
-	{"text": "%s looked through my browser history, only to discover %s.", "pick": 2}
+	{"text": "%s looked through my browser history, only to discover %s.", "pick": 2},
+	{"text": "How to beat your children without leaving any sign of injury"}
 ],
 "answers": [
-	"Imaginary boyfriend",
+	"An imaginary boyfriend",
 	"Frolicking in a meadow",
 	"Your own mother",
 	"Hippies",
 	"Hairy nipples",
 	"Cold pizza and warm beer",
-	"Flamboyant street magician",
-	"Vintage air guitar collection",
+	"A flamboyant street magician",
+	"A vintage air guitar collection",
 	"Zoo animals going rampant and killing their handlers",
 	"Shaving your head to fake having cancer",
 	"Avoid paying taxes",
 	"Driving over the speed limit",
 	"Becoming an organ donor",
-	"Weapons grade plutonium",
+	"Weapons-grade plutonium",
 	"Watching a race between a snail and a turtle",
 	"White powder that is not cocaine",
 	"Caffeine overdose",
-	"Mentally retarded cousin",
+	"A mentally retarded cousin",
 	"Cheesus, the son of Gouda",
 	"Racial segregation",
-	"Deep throating a banana",
+	"Deep-throating a banana",
 	"Extracting heroin from dead junkies",
 	"Faking a smile for a family portrait",
-	"Sexually harassing a co-worker and then finding out she's your cousin",
+	"Sexually harassing a co-worker then finding out she's your cousin",
 	"Used panties",
 	"Violently fisting until your knuckles are bruised",
-	"IRC bot",
+	"An IRC bot",
 	"Premature ejaculation",
-	"A fart that almost wan't a fart",
+	"A fart that almost wasn't a fart",
```

Comment:
There's already a similar card, do we really need this?

"When you fart and a little bit comes out"